lib/OpenQA/Git.pm

```diff
@@ -116,9 +118,14 @@ sub get_current_branch ($self) {
     return trim($r->{stdout});
 }
 
+sub init ($self) {
+    my $r = $self->_run_cmd(['init']);
+    $r->{status} or die $self->_format_git_error($r, 'Failed to init repo');
+}
+
 sub check_sha ($self, $sha) {
-    my $r = $self->_run_cmd(['rev-parse', '--verify', '-q', $sha]);
-    return 0 if $r->{return_code} == 128;
+    my $r = $self->_run_cmd(['rev-parse', '--verify', '-q', $sha], \%CHECK_OPTIONS);
+    return 0 if $r->{return_code} == 1;
```

Comment:
I also checked again this one. This is also 32768, so 128 is the right value

Reply:
For this one I always get `0` in case the sha exists and `1` in case it doesn't and something else if the check didn't work. So I guess `1` is correct here.

Reply:
oh, interesting. the `-q` parameter makes a difference. yeah, it's 1 then.